pocket/chain.json

```
@@ -0,0 +1,112 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "pocket",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.pokt.network/",
+  "pretty_name": "Pocket Network MainNet",
+  "chain_type": "cosmos",
+  "chain_id": "pocket",
+  "bech32_prefix": "pokt",
+  "daemon_name": "pocketd",
+  "node_home": "$HOME/.pocket",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
```

```suggestion
  "slip44": 118,
  "description": "Pocket Network is a decentralized infrastructure protocol that incentivizes a global network of node operators to provide reliable and cost-effective data access for applications across multiple platforms, including blockchains, AI services, and other APIs.",
```

Optional but is included in many networks.